extensions/MYTAditya/CommentPlus.js

```diff
@@ -0,0 +1,372 @@
+          },
+          {
+            opcode: "commentReporterP6",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "[COLOR] // [COMMENT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              COLOR: {
+                type: Scratch.ArgumentType.COLOR,
+                defaultValue: "#037b03",
+              },
+            },
+          },
+          {
+            opcode: "commentBooleanP6",
+            blockType: Scratch.BlockType.BOOLEAN,
+            text: "[COLOR] // [COMMENT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              COLOR: {
+                type: Scratch.ArgumentType.COLOR,
+                defaultValue: "#037b03",
+              },
+            },
+          },
+          {
+            opcode: "commentReporterP7",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "// [COMMENT] [COLOR]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              COLOR: {
+                type: Scratch.ArgumentType.COLOR,
+                defaultValue: "#037b03",
+              },
+            },
+          },
+          {
+            opcode: "commentBooleanP7",
+            blockType: Scratch.BlockType.BOOLEAN,
+            text: "// [COMMENT] [COLOR]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              COLOR: {
+                type: Scratch.ArgumentType.COLOR,
+                defaultValue: "#037b03",
+              },
+            },
+          },
+          {
+            opcode: "commentCPlusPlus",
+            blockType: Scratch.BlockType.CONDITIONAL,
+            branchCount: 2,
+            text: "// [COMMENT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+            },
+          },
+        ],
+      };
+    }
+
+    commentHat() {
+      // no-op
+    }
+
+    commentCommand() {
+      // no-op
+    }
+
+    commentC(args, util) {
+      return true;
+    }
+
+    commentReporter(args) {
+      return args.INPUT;
+    }
+
+    commentBoolean(args) {
+      return args.INPUT || false;
+    }
+
+    commentReporterPPP(args) {
+      return args.INPUT;
+    }
+
+    commentBooleanPPP(args) {
+      return args.INPUT || false;
+    }
+
+    commentReporterPPPP(args) {
+      return args.INPUT || false;
+    }
+
+    commentBooleanPPPP(args) {
+      return args.INPUT;
+    }
+
+    commentBooleanPlus(args) {
+      // no-op
+    }
+
+    commentReporterPlus(args) {
```

Comment:
Please don't mark issues as resolved unless they are **actually** resolved. This still does not return anything. That is a **bug.** Please fix it